### DISCUSSION TITLE
增强开放平台模块 UI 使用体验

### DISCUSSION THREAD

**Discussion by klboke** [apolloconfig/apollo#3744] [Ideas]
目前开发平台管理模块，在 Partal 上只能新建，以及使用明确的第三方应用ID 查询或者更新，缺少列表管理功能，运维管理这块内容会比较困难，目前使用这块功能，只能人工使用笔记记住开通了哪些第三方应用ID，当前UI如：

![image](https://user-images.githubusercontent.com/18591662/120612421-c6362b80-c487-11eb-9bc8-dd27cd6dc12c.png)
 
可以新增一个查看所有的列表管理。

**Comment by nobodyiam** [comment]
Sounds a good idea!

**Comment by klboke** [comment]
realized => https://github.com/apolloconfig/apollo/pull/4436

**Reply by wooln** [reply]
2021年我司devops中使用openapi时就感觉这个管理功能缺失两个功能：1.三方应用列表管理 2.已授权给它的范围查看和移除。最近升级到2.0后发现了 #4436 中已经实现了第一个功能，第二功能若没有特别计划我司可以去PR，继续完善这个大家不太常用的功能。

**Reply by nobodyiam** [reply]
欢迎提交 PR！
